content/_data/changelogs/weekly.yml

```diff
@@ -1654,6 +1654,70 @@
           title: security advisory
         - url: /blog/2017/12/14/security-update/
           title: announcement blog post
+- version: "2.96"
+  date: 2017-12-17
+  changes:
+    - type: major bug
+      pull: 3193
+      issue: 48365
+      message: >
+        Make sure detached plugins (plugins whose functionality used to be part of Jenkins itself) are installed when upgrading Jenkins past the version at which the plugin was detached.
+
```

Comment:
hm? empty line

Reply:
Grouping by kind for readability, since it was so many changes this time.